oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneDocumentMaker.java

```diff
@@ -54,6 +55,10 @@
 public class LuceneDocumentMaker extends FulltextDocumentMaker<Document> {
     private static final Logger log = LoggerFactory.getLogger(LuceneDocumentMaker.class);
 
+    private static final String DYNAMIC_BOOST_TAG_NAME = "name";
+    private static final String DYNAMIC_BOOST_TAG_CONFIDENCE = "confidence";
+    private static final String DYNAMIC_BOOST_SPLIT_REGEX = "[:/]";
```

Comment:
Would It make sense to get these 3 properties DYNAMIC_BOOST_TAG_NAME, DYNAMIC_BOOST_TAG_CONFIDENCE and DYNAMIC_BOOST_SPLIT_REGEX  from the index definition ? (This would make this more generic in case these nodes are generated by some service other than the dynamic tag one - that service could have a different property name for a value that can be used as boost.)

Reply:
Sure it is possible. However, the algorithm is so specific anyway, I don't think we can reuse it ever. So I see no point in making it configurable: it would just add complexity, without solving anything. So I'm against that :-)